src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -384,4 +384,30 @@ public void setCreated_NonProduction(Date createdAt) {
     public void setUpdatedAt_NonProduction(Date updatedAt) {
         this.updatedAt = updatedAt;
     }
+
+    /**
+     * Checks whether the edit form of student has changed the section value.
+     * @param originalStudentAttribute
+     * @return true if either section has changed from previous values. 
+     */
+    public boolean hasSectionChanged(StudentAttributes originalStudentAttribute) {
+       if (((this.section!=null) && !(this.section.equals(originalStudentAttribute.section)))) {
```

Comment:
You could return `(this.section != null) && !(this.section.equals(originalStudentAttribute.section))` and omit the if else statement. Also take note to ensure spaces around the boolean operator i.e. `(this.section != null)`